### ISSUE TITLE
crash with unsupported MediaType

### ISSUE THREAD

**Issue by r0bj** [flant/k8s-image-availability-exporter#8]
`k8s-image-availability-exporter` is crashing with:
```
time="2020-05-29T18:56:11Z" level=error msg="unsupported MediaType: \"application/vnd.docker.distribution.manifest.v1+prettyjws\", see https://github.com/google/go-containerregistry/issues/377" availability_mode=registry_v1_api_not_supported image_name="quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.25.1"
```
and
```
time="2020-05-29T18:53:26Z" level=error msg="unsupported MediaType: \"application/vnd.docker.distribution.manifest.v1+prettyjws\", see https://github.com/google/go-containerregistry/issues/377" availability_mode=registry_v1_api_not_supported image_name="quay.io/k8scsi/csi-node-driver-registrar:v1.2.0"
```

Eventually pod is in `CrashLoopBackOff`.

**Comment by zuzzas** [comment]
Hi. Sorry for the delay.

Could you provide a `kubectl describe` of this Pod? Perhaps a Pod is crashing on the livenessProbe?

**Comment by r0bj** [comment]
@zuzzas It seems that it was only issue with version 0.1.5 (from example deployment https://github.com/flant/k8s-image-availability-exporter/blob/master/deploy/deployment.yaml). Latest available version 0.1.13 works fine. Thanks.
